pkg/cpu/cpu.go

```diff
@@ -141,3 +160,54 @@ func GetLoad() Load {
 		Error: "",
 	}
 }
+
+//GetCPUHardware fetches the Hardware info
+func GetCPUHardware() HardwareCPU {
+
+	cpuHardwareInfo := HardwareCPU{}
```

Comment:
We shouldn't need to use lspcu we should simply use https://github.com/klauspost/cpuid

Reply:
It is x86 only, though.

Reply:
https://github.com/shirou/gopsutil is another alternative for all platforms.